ls1intum/Artemis
# Bugfix guided tour: Only trigger cancel tour step if tour step component is active and further bug fixes

### Checklist

### Motivation and Context
It is possible to trigger the `cancel-tour-step` with the `ESC` key even if the tour is not active (see screenshot 1) and also displays `Start tutorial` instead of `Continue tutorial`. This PR fixes this issue.

I also fixed further minor issues with this PR: 
- The resize tour was displayed on mobile devices - this is now prevented (screenshot 2)
- The Java tour step had a hint that has to be removed (screenshot 3)
- The course administration table had the wrong translation key for the `presentation score` (screenshot 4)

### Steps for Testing
1. Log in to Artemis
2. Navigate anywhere
3. Click `esc` key
4. Cancel tour should not be displayed

### Screenshots
Screenshot 1
![image](https://user-images.githubusercontent.com/33764166/66230933-913b6980-e6e5-11e9-8bd4-9fac4c1b2f55.png)
Screenshot 2
![IMG_0032](https://user-images.githubusercontent.com/33764166/66261686-5d8a3d80-e7d2-11e9-95d1-fab64f1d2c10.PNG)
Screenshot 3
![image](https://user-images.githubusercontent.com/33764166/66259505-0bd1bb00-e7b2-11e9-9bde-281c41e78e44.png)
Screenshot 4
![image](https://user-images.githubusercontent.com/33764166/66261690-6e3ab380-e7d2-11e9-9c44-7e27d6380b63.png)
